package.json

```diff
@@ -121,12 +120,9 @@
     "react-table": "6.8.6",
     "redux": "^4.0.0",
     "redux-batched-subscribe": "^0.1.6",
-    "redux-devtools-extension": "^2.13.5",
     "redux-logger": "^3.0.6",
     "redux-observable": "^0.18.0",
     "rxjs": "^5.5.10",
-    "slate": "0.40.2",
```

Comment:
Module not found: Error: Can't resolve 'slate' in '/Users/conorpirie/projects/genny/alyson-v7/node_modules/slate-base64-serializer/lib'

Reply:
Module not found: Error: Can't resolve 'slate-html-serializer' in '/Users/conorpirie/projects/genny/alyson-v7/src/views/components-legacy/input/rich-text-editor'

Reply:
#24 will fix this issue, that can be merged first and then this. I have got rid of these packages on that PR.